test/integration/apiserver/apply/status_test.go

```diff
@@ -199,7 +206,7 @@ func TestApplyStatus(t *testing.T) {
 				if !findManager(managedFields, "apply_status_test") {
 					t.Fatalf("Couldn't find apply_status_test: %v", managedFields)
 				}
-				if !findManager(managedFields, "create_test") {
+				if _, ok := ignoreCreateManagementList[mapping.Resource]; !findManager(managedFields, "create_test") && !ok {
 					t.Fatalf("Couldn't find create_test: %v", managedFields)
 				}
```

Comment:
make this both a positive and negative test to ensure we don't have stale items in the ignoreCreateManagementList list:

```
if !findManager(managedFields, "create_test") {
  if !ignoreCreateManagementList[mapping.Resource] {
    t.Fatalf("Couldn't find create_test: %v", managedFields)
  }
} else if ignoreCreateManagementList[mapping.Resource] {
  t.Fatalf("found create_test in ignoreCreateManagementList resource: %v", managedFields)
}
```